v3-certificate/generate-certificate

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
```

Comment:
Is this executable? Can't see a way to see that in the web interface.

Reply:
Yep